net-firewall/ferm/ferm-2.3.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit eutils systemd
+
+DESCRIPTION="Command line util for managing firewall rules"
+HOMEPAGE="http://ferm.foo-projects.org/"
+SRC_URI="http://ferm.foo-projects.org/download/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2"
```

Comment:
The license can be GPLv2-only or GPLv2+. COPYING wouldn't tell you that. It's better to check the source files themselves for this.